### PR TITLE
Container Services

### PR DESCRIPTION
**Pull request by ckleban**
Add Container Services and cabernets getting started guide

### CODE REVIEW

File: Container Services/getting-started-with-kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+
+### Clusters of VMs or Physical Servers, your choice.
+
+- We support Kubernetes clusters on both Virtual Machines or Physical Servers. If you want to use physical servers for the worker nodes (minions), simple use the --minion_type=bareMetal flag.
+- For more information on pyhsical servers, visit: https://www.ctl.io/bare-metal/)
+- Physical serves are only available in the VA1 and GB3 data centers.
+- VMs are available in all 13 of our public cloud locations
+
+
+## Requirements
```
Comment (jim-phillips) [inline Container Services/getting-started-with-kubernetes.md]: change to `###`

File: Container Services/getting-started-with-kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+And 
+
+- A linux host with the following items installed:
+- ansible _version 2.0_ or newer.  
+- python 
+- pip
+- git
+
+## Script Installation
```
Comment (jim-phillips) [inline Container Services/getting-started-with-kubernetes.md]: change to `###`

File: Container Services/getting-started-with-kubernetes.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+  cd adm-kubernetes-on-clc/
+  pip install -r requirements.txt
+
+  # getting started
+  cd ansible
+  cp credentials.sh.template credentials.sh; vi credentials.sh
+  source credentials.sh
+```
+
+## Cluster Creation
```
Comment (jim-phillips) [inline Container Services/getting-started-with-kubernetes.md]: change to `###`

File: Container Services/getting-started-with-kubernetes.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+You will need the kubectl client program to manage your kubernetes cluster. You can access this by one of two ways:
+
+- The master server has this installed. SSH to the master server and run kubectl from there. 
+- Use your local machine (or some remote machine to the cluster)
+
+#### Network access to the cluster
+- Network access from a remote box to the master can be achieved by using the VPN access that you needed in the previous step or by adding a public IP address to the master server. 
+
+
+## Kubernetes Usage
```
Comment (jim-phillips) [inline Container Services/getting-started-with-kubernetes.md]: `###`

File: Container Services/getting-started-with-kubernetes.md [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+
+
+## What Kubernetes features do not work on CenturyLink Cloud
+
+- At this time, there is no support services of the type 'loadbalancer'. We are actively working on this and hope to publish the changes soon. 
+- At this time, there is no support for persistent storage volumes provided by CenturyLink Cloud. However, customers can bring their pwn persistent storage offering.
+
+
+
+## Postrequisite
```
Comment (jim-phillips) [inline Container Services/getting-started-with-kubernetes.md]: `###`

File: Container Services/getting-started-with-kubernetes.md [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+```
+  bash kube-up.sh --clc_cluster_name=k8s_3 --minion_type=standard --minion_count=10 --datacenter=VA1 -mem=6 -cpu=4
+```
+
+## Cluster Deletion
+
+To delete a cluster, log into the CenturyLink Cloud control portal and delete the parent server group that contains the Kubernetes Cluster. We hope to add a scripted option to do this soon. 
+
+
+## What Kubernetes features do not work on CenturyLink Cloud
```
Comment (jim-phillips) [inline Container Services/getting-started-with-kubernetes.md]: `###`

File: Container Services/getting-started-with-kubernetes.md [hunk 7]
```diff
@@ -0,0 +1,210 @@
...
+ bash kube-up.sh --clc_cluster_name=k8s_2 --minion_type=standard --minion_count=6 --datacenter=VA1 --etcd_separate_cluster=yes
+```
+Create a cluster with name of k8s_3, 1 master node, and 10 worker minions (on VMs) with higher mem/cpu, in UC1:
+
+
+```
+  bash kube-up.sh --clc_cluster_name=k8s_3 --minion_type=standard --minion_count=10 --datacenter=VA1 -mem=6 -cpu=4
+```
+
+## Cluster Deletion
```
Comment (jim-phillips) [inline Container Services/getting-started-with-kubernetes.md]: `###`

File: Container Services/getting-started-with-kubernetes.md [hunk 8]
```diff
@@ -0,0 +1,210 @@
...
+
+Goto the following url in a web browser that has access to the master server:
+
+http://(IP-ADDRESS-OF-MASTER-SERVER):8080/api/v1/proxy/namespaces/kube-system/services/kube-ui
+
+#### How to use the kubectl client
+
+To get your Kubernetes cluster's information:
+
+``` kubectl -s http://(IP-ADDRESS-OF-MASTER-SERVER):8080 cluster-info ```
```
Comment (jim-phillips) [inline Container Services/getting-started-with-kubernetes.md]: either move the text to a new lines or change the triple tic to single.

Do the same for all below.

**Comment by jim-phillips** [comment]
@ckleban Please address comments

**Comment by ckleban** [comment]
Thanks Jim for the review. I've addressed all of your comments.

**Pull request merged**